src/css/external/jquery.floatingscroll.css

```diff
@@ -0,0 +1,19 @@
+.fl-scrolls, .fl-scrolls div {
```

Comment:
This plugin provide a floating scroll, scroll bar will always visible, so that user dont have to go to the end of page to reach the horizontal scrollbar